news/2020-02-03-monthly-beatmapping-contest-february.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "Monthly Beatmapping Contest: February"
+date: 2020-02-03 02:00:00 +0000
+---
+
+<!-- todo
+- replace "top XX" with actual value
+- doublecheck contest id
+-->
+
+New month? New mapping contest! Vote on last month's submissions and compete to become the next Elite Mapper!
+
+![monthly beatmapping contest banner](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+The Monthly Beatmapping Contest (MBC) is back after a multi-year hiatus. Check out [last month's contest announcement](https://osu.ppy.sh/home/news/2020-01-05-monthly-beatmapping-contests-return) if you need to catch up.
+
+January's contest received a *massive* turnout of over 100 valid submissions! A group of experienced beatmappers narrowed them down to the top XX — the best of the best — so we encourage you to download those and vote for your favorite.
+
+**Vote for last month's contest [here](https://osu.ppy.sh/community/contests/84)!** Results will be published in 2 weeks.
+
+Now that housekeeping is out of the way, let's talk about the contest this month!
+
+## Schedule
+
+As per usual, the timeline for this month's contest is as follows:
+- **Beatmapping (3 weeks):** Contest is announced and beatmappers create their entries.
+- **Screening (1 week):** Entries are evaluated by experienced beatmappers.
+- **Voting (2 weeks):** Top entries from screening are published for the osu! community to vote on. This overlaps with the next month's beatmapping portion.
+- **Rewards:** Winners are announced and prizes are redeemed, including profile badges, osu!supporter, and the "Elite Mapper" user title!
+
+If you're curious about the details behind any of these, dive into the [Monthly Beatmapping Contest wiki article](/wiki/Contests/Monthly_Beatmapping_Contest). It answers every question you could ask. Probably.
+
+## Beatmapping
+
+### <a id="limitation"></a>Limitation
+
+Each month's contest has its own mapping limitation that participants must design their maps around. For example, last month's limitation was "straight slider shapes only", so sliders in all beatmap submissions were... straight lines. Pretty simple, right?
+
+This month's limitation: **star rating cannot exceed 4.** 
```

Comment:
can spesify by saying "star rating cannot exceed 4.00" as people can see "4." as 4.99* instead

Reply:
It's just a period, I don't think people would misunderstand that :/

Reply:
> [Random mapper] Today at 9:23 PM
> 4.00 stars or 4.99 stars ?
it's more of a thing where you either include the section of the 4 to 4.99 or make it max to 4.00

Reply:
yes i think 4.00* would be more clear